src/Tinkoff.ISA.Core/KnowledgeRequest.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Tinkoff.ISA.Core
+{
+    public class KnowledgeRequest
+    {
+        public DateTimeOffset StartDate { get; set; }
+
+        public int StartAt { get; set; }
```

Comment:
Yeah, you right.
The potential case that may occur consist of the following: when we ask for documents starting from, for instance, 26.06.2019 21:45 and there are lots of documents with such a date (update or create, doesn't really matter), so the date of the last document in returned batch won't be changed. We end up with an endless loop if don't specify additional `startAt` pointer.

Reply:
But if we consider to make this code more generic maybe it will be better to make this property generic, because in some other document it can be string or event other type. What you think about it?